test/fixtures/account/cryptos.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the "{}" from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
+# column: value
+#
+two: {}
+# column: value
```

Comment:
These are the cause of the CI test failure.  Since an `Accountable` must always be associated with an `Account`, these create orphaned records that break the account rollups.